src/20241106_Multi_GHOAvaxLaunch/AaveV3Arbitrum_GHOAvaxLaunch_20241106.t.sol

```diff
@@ -0,0 +1,500 @@
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+import {GovernanceV3Avalanche} from 'aave-address-book/GovernanceV3Avalanche.sol';
+import {MiscAvalanche} from 'aave-address-book/MiscAvalanche.sol';
+import {UpgradeableGhoToken} from 'gho-core/gho/UpgradeableGhoToken.sol';
+import {IUpgradeableTokenPool_1_4} from 'src/interfaces/ccip/IUpgradeableTokenPool_1_4.sol';
+import {AaveV3Arbitrum_GHOAvaxLaunch_20241106} from './AaveV3Arbitrum_GHOAvaxLaunch_20241106.sol';
+import {AaveV3Avalanche_GHOAvaxLaunch_20241106} from './AaveV3Avalanche_GHOAvaxLaunch_20241106.sol';
+
+/**
+ * @dev Test for AaveV3Arbitrum_GHOAvaxLaunch_20241106
+ * command: FOUNDRY_PROFILE=arbitrum forge test --match-path=src/20241106_Multi_GHOAvaxLaunch/AaveV3Arbitrum_GHOAvaxLaunch_20241106.t.sol -vv
+ */
+contract AaveV3Arbitrum_GHOAvaxLaunch_20241106_Test is ProtocolV3TestBase {
+  AaveV3Arbitrum_GHOAvaxLaunch_20241106 internal proposal;
+
+  UpgradeableBurnMintTokenPool public constant TOKEN_POOL =
+    UpgradeableBurnMintTokenPool(MiscArbitrum.GHO_CCIP_TOKEN_POOL);
+  address public constant GHO_TOKEN = AaveV3ArbitrumAssets.GHO_UNDERLYING;
+  UpgradeableGhoToken public GHO = UpgradeableGhoToken(GHO_TOKEN);
+
+  address public constant TOKEN_ADMIN_REGISTRY = 0x39AE1032cF4B334a1Ed41cdD0833bdD7c7E7751E;
+  address public constant REGISTRY_ADMIN = 0x8a89770722c84B60cE02989Aedb22Ac4791F8C7f;
+  address public constant AVAX_GHO_TOKEN = 0x2e234DAe75C793f67A35089C9d99245E1C58470b;
+  address public constant AVAX_TOKEN_POOL = 0x5991A2dF15A8F6A256D3Ec51E99254Cd3fb576A9;
+  address public constant AVAX_REGISTRY_ADMIN = 0xA3f32a07CCd8569f49cf350D4e61C016CA484644;
+  address public constant AVAX_TOKEN_ADMIN_REGISTRY = 0xc8df5D618c6a59Cc6A311E96a39450381001464F;
+  address public constant AVAX_RMN_PROXY = 0xcBD48A8eB077381c3c4Eb36b402d7283aB2b11Bc;
+  address public constant AVAX_ROUTER = 0xF4c7E640EdA248ef95972845a62bdC74237805dB;
+  address public constant ETH_TOKEN_POOL = MiscEthereum.GHO_CCIP_TOKEN_POOL;
+  address public constant CCIP_ARB_ETH_ON_RAMP = 0x67761742ac8A21Ec4D76CA18cbd701e5A6F3Bef3;
+  address public constant CCIP_ARB_ETH_OFF_RAMP = 0x91e46cc5590A4B9182e47f40006140A7077Dec31;
+  address public constant CCIP_ARB_AVAX_ON_RAMP = 0xe80cC83B895ada027b722b78949b296Bd1fC5639;
+  address public constant CCIP_ARB_AVAX_OFF_RAMP = 0x95095007d5Cc3E7517A1A03c9e228adA5D0bc376;
+  address public constant TOKEN_POOL_AND_PROXY = 0x26329558f08cbb40d6a4CCA0E0C67b29D64A8c50;
+  uint64 public constant CCIP_AVAX_CHAIN_SELECTOR = 6433500567565415381;
+
+  event Minted(address indexed sender, address indexed recipient, uint256 amount);
+  event Burned(address indexed sender, uint256 amount);
+  event Transfer(address indexed from, address indexed to, uint256 value);
+
+  function setUp() public {
+    // Execute Avax proposal to deploy Avax token pool
+    vm.createSelectFork(vm.rpcUrl('avalanche'), 53559217);
```

Comment:
there is no need to create an avalanche fork and deploy contracts there for arbitrum tests (especially because you don't seem to re-use this fork later in any of the tests as well) - pls save this for separate E2E tests where we switch between forks. for avalanche contracts (token pool, gho) can simply use `makeAddr` for these chain specific tests